Library/Homebrew/cli/args.rb

```diff
@@ -133,6 +133,11 @@ def context
         Context::ContextStruct.new(debug: debug?, quiet: quiet?, verbose: verbose?)
       end
 
+      def only_path_formula_or_cask
```

Comment:
```suggestion
      def only_formula_or_cask
```

Not sure why `path` should be in the name.